api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -433,8 +433,33 @@ type NodeAgentConfig struct {
 	// Embedding NodeAgentConfigMapSettings
 	// +optional
 	NodeAgentConfigMapSettings `json:",inline"`
+	// Embedding KopiaRepoOptions
+	// +optional
+	KopiaRepoOptions `json:",inline"`
+}
+
+type KopiaRepoOptions struct {
```

Comment:
the idea to have a separate struct is because this would be datamover options and not FSB options? (I think I heard some discussion around this)

Reply:
I think separate to use marshal:
https://github.com/openshift/oadp-operator/pull/1705/files/2e2b78de35561c27a6dd42af95d88e012eb879a2#diff-3d7a2c405414ff19b6660300351bf73aa3c836948cfe76a80f4e9e4d7b689d07R31

Reply:
alright